frontend/src/components/dashboard/Analysis.tsx

```diff
@@ -47,14 +47,16 @@ import {
   AnalyzerStateAction,
   AnalyzerStateActionType,
 } from "./Analyzer.state";
+import { SelectPlugin, SplitPlugin, SplitType } from "./chartjs-plugins";
 
 type AdditionalChartData = {
   groups: {
     key: string;
     name: string;
     analyses: CurrentAnalysis[];
   }[];
-  isSelected: boolean;
+  isSelectedForComparison: boolean;
+  isManuallySelected: boolean;
```

Comment:
They're both manually selected, I'm wondering if we could find better naming... but I have no ideas.